lms/djangoapps/support/static/support/jsx/entitlements/entitlements.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export function EntitlementSupportPage() {
```

Comment:
I am unsure what the full directory structure is supposed to look like but is there a reason why the filename is `entitlements/entitlements.jsx` vs. `entitlements/index.jsx`? `index.jsx` is generally the entry filename pattern for a given directory (at least it seems that way at edX).

Reply:
I'll probably switch this to index, though if you checkout the [webpack config](https://github.com/edx/edx-platform/blob/07208acf7d150c031efcc6428410f693f9dd4b0e/webpack.common.config.js#L27) it looks like we don't have much of a naming standard in platform, most of these do something like naming the file the same things as the component.